pkg/timestream/executor.go

```diff
@@ -17,19 +18,18 @@ func ExecuteQuery(ctx context.Context, query models.QueryModel, runner queryRunn
 
 	raw, err := Interpolate(query, settings)
 	if err != nil {
-		dr.Error = err
-		return
+		return errorsource.Response(err)
```

Comment:
we probably shouldn't mix return types (i don't really like named return variables so i'm fine with just removing the dr from the function declaration)

Reply:
Ah, totally - I got rid of one but not the other. (I also don't love named returns except for the cool things you can do with them with `defer`)